src/pages/gen2/start/quickstart/nextjs-app-router/index.mdx

```diff
@@ -13,292 +13,17 @@ export function getStaticProps(context) {
 
 This Quickstart guide will walk you through how to build a task list application with TypeScript, Next.js **App Router**, and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Next.js](https://nextjs.org/docs/getting-started/installation), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
 
-## Prerequisites
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+import createProject from 'src/fragments/gen2/quickstart/create-nextjs-project.mdx';
```

Comment:
@kevinold I wonder if we should co-locate the src address with where the fragment is used. We use this convention on other docs so authors can quickly find where the fragment is filed without having to scroll back up. @reesscot is there a preference from the docs team on this format?

Reply:
@heatheramz No problem, I'll updated if that is the preference.

Reply:
Done.